service/src/test/java/bio/terra/catalog/api/VersionTest.java

```diff
@@ -0,0 +1,56 @@
+package bio.terra.catalog.api;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import bio.terra.catalog.config.VersionConfiguration;
+import bio.terra.catalog.service.CatalogStatusService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+
+@TestConfiguration
+class TestVersionConfig {
+  @Bean
+  @Primary
+  public VersionConfiguration versionConfiguration() {
+    VersionConfiguration mockConfig = mock(VersionConfiguration.class);
+    when(mockConfig.getGitTag()).thenReturn("0.1.0");
+    when(mockConfig.getGitHash()).thenReturn("hash");
+    when(mockConfig.getGithub()).thenReturn("https://github.com/");
+    when(mockConfig.getBuild()).thenReturn("build");
+    return mockConfig;
+  }
+}
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {PublicApiController.class, TestVersionConfig.class})
+@WebMvcTest
+public class VersionTest {
```

Comment:
A minor point; JUnit 5 doesn't require test classes or methods to be `public` like 4 did. So the junit5 recommendation is to use default access for classes and methods. https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods